ppr-ui/src/composables/useInputRules.ts

```diff
@@ -19,9 +25,13 @@ export const useInputRules = () => {
     ]
   }
 
-  const isNumber = (): Array<Function> => {
+  const isNumber = (numberType: string, maxDigits: number = null, maxValue: number = null): Array<Function> => {
+    const maxDigitRule = new RegExp(`^\\d{1,${maxDigits}}$`)
+
     return [
-      v => (v ? /^\d+$/g.test(v) : true) || 'Invalid number'
+      v => (v ? /^\d+$/g.test(v) : true) || `${numberType} must be a valid whole number (cannot contain decimals)`,
+      v => ((v && maxDigits) ? maxDigitRule.test(v) : true) || `Maximum ${maxDigits} characters`,
```

Comment:
You could do this two ways. Count the digits, as above, or write logic to cast the number to a string, as `.length` is a property of a `string` instance. 
Since it's a number i went with counting digits.

Reply:
Sounds good, I'm good either way. For _maxLength_ rule and official doc we can do this also:
`v => (v || '').length <= #`

Reply:
For sure, i guess it boils down to preference or readability. 
Frankly, i think it's quite easy to read as is. 
`v => condition || fallback`

Reply:
The `v || ' '` might be a better way to handle short circuiting though!